presto-tests/src/main/java/com/facebook/presto/tests/H2QueryRunner.java

```diff
@@ -301,6 +303,15 @@ else if (type instanceof DecimalType) {
                             row.add(decimalValue);
                         }
                     }
+                    else if (type instanceof ArrayType) {
+                        Object[] arrayValue = (Object[]) resultSet.getArray(i).getArray();
+                        if (resultSet.wasNull()) {
+                            row.add(null);
+                        }
+                        else {
+                            row.add(new ArrayList<>(asList(arrayValue)));
```

Comment:
Nulls are sad. You can slightly simplify though with Guava `Lists.newArrayList(E...)`